### DISCUSSION TITLE
Pretrained model refuses to converge.

### DISCUSSION THREAD

**Discussion by MostafaAhmed98** [NVIDIA-NeMo/NeMo#9047] [Q&A]
Hello,

Im trying to build an Arabic asr with common voice dataset with pretrained stt_en_citrinet_256 and after a 120 epochs i found that my model didn't improve and still making a bad WER (0.95) on test set.

i used sub-word sentencepiece tokenizer with 4096 size and changed the pretrained model vocab, tokenizer, train and test config as below:

`from omegaconf import OmegaConf, open_dict`
`params = OmegaConf.load("./configs/config_bpe.yaml")`
`params.model.train_ds.manifest_filepath = 'train.json'`
`params.model.validation_ds.manifest_filepath = 'dev.json'`
`first_asr_model.change_vocabulary(
    new_tokenizer_dir='tokenizer_spe_unigram_v4096',
    new_tokenizer_type="bpe"
)`
`first_asr_model.setup_training_data(train_data_config=params['model']['train_ds'])`
`first_asr_model.setup_validation_data(val_data_config=params['model']['validation_ds'])`

below is my train, val loss and WER

![image](https://github.com/NVIDIA/NeMo/assets/90983988/c646f478-4a19-4ce6-8141-b86f9e7a29f3)
![image](https://github.com/NVIDIA/NeMo/assets/90983988/968df032-5c3d-4d44-8ef9-09dbcfc034aa)

**Comment by Houss3m** [comment]
Hi Ahmed, were you able to fix it?
